aodncore/pipeline/handlerbase.py

```diff
@@ -1022,19 +1022,15 @@ def postprocess(self):  # pragma: no cover
     # "public" methods
     #
 
-    def download_remotepipelinefilecollection(self, remotepipelinefilecollection, local_path=None):
-        """Helper method to download a RemotePipelineFileCollection or RemotePipelineFile using the handler's internal
-            storage broker
+    def add_pipelinefile(self, pipeline_file):
```

Comment:
Good idea, added in the latest commit.

Reply:
This is pretty nice for child handlers actually, because you can add a path with a single statement now,

`self.add_to_collection(product_path, publish_type=PipelineFilePublishType.HARVEST_UPLOAD)`